apprise/plugins/NotifyPushover.py

```diff
@@ -240,6 +254,34 @@ def __init__(self, token, targets=None, priority=None, sound=None,
         else:
             self.priority = priority
 
+        # The following are for emergency alerts
+        if self.priority == PushoverPriority.EMERGENCY:
+
+            # How often to resend notification, in seconds
+            self.retry = NotifyPushover.template_args['retry']['default']
```

Comment:
This is a clever way of doing this. I should probably adapt some of my other code to do this too and avoid re-entering the same content everywhere.  Nice!